pkg/network/tracer/ebpf_conntracker_test.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+// +build linux_bpf
+
+package tracer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/network/tracer/offsetguess"
+)
+
+func TestEbpfConntrackerLoadTriggersOffsetGuessing(t *testing.T) {
+	offsetguess.TracerOffsets.Reset()
+
+	cfg := testConfig()
+	cfg.EnableRuntimeCompiler = false
```

Comment:
should this also test that offsets are empty when runtime compilation is enabled? or maybe that is tested elsewhere

Reply:
Tested here: https://github.com/DataDog/datadog-agent/pull/16899/files#diff-cc699a7d4964703987a4ad03a9e86e2c2f1510c2d894ead2bf7776311446203eR232